src/sections/Community/Members-grid/membersGrid.style.js

```diff
@@ -89,10 +102,20 @@ export const selectStyles = {
   input: styles => ({ ...styles, ...dot(), zIndex: 900 }),
   placeholder: styles => ({ ...styles, ...dot(), zIndex: 900 }),
   singleValue: (styles, { data }) => (data.value === "all"
-    ? { ...styles, zIndex: 900 }
-    : data.value === "maintainers" ? {...styles, ...img(icon5)}
-      : data.value === "meshmates" ? { ...styles, ...img(meshmateIcon) }
-        : { ...styles, ...dot(data.color), zIndex: 900 }),
+    ? { ...styles, zIndex: 900}
+    : data.value === "maintainers" ? {...styles, ...img(data.icon),backgroundPosition: "30% 50%"}
```

Comment:
Same as above. It would also be nice to do the same with `backgroundPosition`.

Reply:
A separate question - What happens if the same `backgroundPosition` is kept?

Reply:
I thought it looked a little cramped up with the text, but I agree this is too redundant. I'll do something about this.